source/index.html.md

```diff
@@ -1005,6 +1005,112 @@ supported.
   Also, <code>initiator</code> cannot be <code>merchant</code>.
 </p>
 
+##### Method: `token`
+
+Mastercard and Visa supply and require different sets of parameters for MDES and
+VTS transactions, respectively. Only one scheme can be used for a given 
+authorization.
+
+<dl class="dl-vertical">
+  <dt>
+    token[visa]
+    <span class="type">dictionary</span>
+  </dt>
+  <dd>
+    Visa token data.
+    <div class="type">Optional. Cannot be present if <code>[mastercard]</code> is present.</div>
+  </dd>
+  <dt>
+    token[mastercard]
+    <span class="type">dictionary</span>
+  </dt>
+  <dd>
+    Mastercard token data.
+    <div class="type">Optional. Cannot be present if <code>[visa]</code> is present.</div>
+  </dd>
+</dl>
+
+##### Method: `token[mastercard]`
+
+<dl class="dl-vertical">
+  <dt>token[mastercard][tan]
+    <span class="type">[0-9]{12,19}</span>
+  </dt>
+  <dd>
+    Token Account Number to charge.
+    Found in <code>encryptedPayload.encryptedData.accountNumber</code> in the MDES response.
+  </dd>
+  <dt>token[mastercard][expiry]
+    <span class="type">[0-9]{6}</span>
+  </dt>
+  <dd>
+    Expiry.
+    Found in <code>encryptedPayload.encryptedData.applicationExpiryDate</code> in the MDES response.
+  </dd>
+  <dt>token[mastercard][cryptogram]
+    <span class="type">[:hex:]</span>
```

Comment:
I am pretty sure this ought to be base64 rather than hex.

Reply:
Indeed.

Reply:
```suggestion
    <span class="type">[:base64:]</span>
```